sample_apps/run_example.sh

```diff
@@ -681,9 +681,7 @@ function rm_non_git_files() {
     fi
 
     local tmp_islet_dir="/tmp/islet-save"
-    if [ ! -d "${tmp_islet_dir}" ]; then
-        mkdir "${tmp_islet_dir}"
-    fi
```

Comment:
Can you please retain this `if()` check I added?

Normally, in a clean dev-env you should not need this as this dir will be deleted in normal execution flow.

What happened in this triage case is that due to the failure of the Islet test-machinery, this dir used to be left behind. So, upon a re-run of this script, it would fail complaining that the dir already exists.

There possibly is a better place to "fix" this, as a cleanup of the exception 'handling' caught by this bash script. But figuring that out, I felt, is a bit too much work.

Hence, I applied this "Patch-fix" to cope with a dir already existing. If you think using a left-over dir could be problematic, one thing we could do in the `else` is to `rm -rf "${tmp_islet_dir}/*` to clean-up stale contents.

I am not hung-up on this too much, but some fix to allow for a tmp-dir to be around may improve robustness of this script, overall. 

Just a thought ... figure out what seems to make best sense, and do something reasonable.

Reply:
Another approach is to delete the dir if it exists, and create a new one. Something like:

```
if [  -d "${tmp_islet_dir}" ]; then
        rmdir -rf "${tmp_islet_dir}"
fi
```

I think this will both (a) cope with the left-over-dir issue I ran into and will (b) avoid re-using stale dir contents.

Reply:
I restored it :)